diff --git ga-5.7.2/armci/src/include/armci.h ga-5.7.2/armci/src/include/armci.h
index 62de274f..fa7dfcdf 100644
--- ga-5.7.2/armci/src/include/armci.h
+++ ga-5.7.2/armci/src/include/armci.h
@@ -374,8 +374,8 @@ extern void ARMCI_SET_AGGREGATE_HANDLE(armci_hdl_t* nb_handle);
 
 extern void ARMCI_UNSET_AGGREGATE_HANDLE(armci_hdl_t* nb_handle);
 
-#define ARMCI_INIT_HANDLE(hdl) do {((double *)((hdl)->data))[0]=0; \
-  ((double *)((hdl)->data))[1]=0; }while(0)
+#define ARMCI_INIT_HANDLE(hdl) do {((int *)((hdl)->data))[0]=0; \
+  ((int *)((hdl)->data))[1]=0; }while(0)
 
 /* -------------- ARMCI Non-collective memory allocator ------------- */
 typedef struct armci_meminfo_ds {
