--- a/ma/ma.c	2014-01-27 09:02:07.509611036 -0800
+++ b/ma/ma.c	2014-01-27 09:04:20.575921035 -0800
@@ -1376,7 +1376,7 @@
             /* print preamble if necessary */
             if (first_bad_block && (preamble != (char *)NULL))
             {
-                (void)printf(preamble);
+                (void)printf("%s", preamble);
                 first_bad_block = MA_FALSE;
             }
 
@@ -1400,7 +1400,7 @@
             /* print preamble if necessary */
             if (first_bad_block && (preamble != (char *)NULL))
             {
-                (void)printf(preamble);
+                (void)printf("%s", preamble);
                 first_bad_block = MA_FALSE;
             }
 
@@ -1425,7 +1425,7 @@
             /* print preamble if necessary */
             if (first_bad_block && (preamble != (char *)NULL))
             {
-                (void)printf(preamble);
+                (void)printf("%s", preamble);
                 first_bad_block = MA_FALSE;
             }
 
--- a/global/src/base.h	2014-01-27 09:20:13.764412896 -0800
+++ b/global/src/base.h	2014-01-27 09:20:39.475493288 -0800
@@ -294,7 +294,7 @@
   int _d, _l;                                                        \
   char *str= "cannot locate region: ";                               \
   char err_string[ERR_STR_LEN];                                      \
-  sprintf(err_string, str);                                          \
+  sprintf(err_string, "%s", str);                                    \
   _d=0;                                                              \
   _l = strlen(str);                                                  \
   sprintf(err_string+_l, "[%ld:%ld ",(long)lo[_d],(long)hi[_d]);     \
